modules/ROOT/pages/enable-inbound-traffic.adoc

```diff
@@ -110,8 +122,18 @@ If you do not see *Add TLS Context*, ensure you are viewing your secret group in
 . After saving your TLS context, click *Secret Groups*, then click the *Finish*. This enables TLS Context for your *Runtime Fabric.*
 
 == Enable Inbound Traffic
+The Runtime Fabric internal load balancer supports OpenSSL 1.1.1 and TLS 1.3, which provide the following security enhancements and performance improvements over TLS 1.2:
+ 
+* 2x or greater TLS 1.2 Connection Performance Throughput Boost.
+* Reduction of one round trip in full handshake for TLS 1.3 vs. TLS 1.2.
+* TLS 1.2 and TLS 1.3 ChaCha20-Poly1305 ciphers for better mobile and IoT device support.
+* TLS 1.3 Protection against downgrade attacks.
+* Continued support for both RSA and ECDSA keys.
+* TLS 1.3 removes obsolete and insecure features from TLS 1.2 and TLS 1.1. Refer to xref:release-notes::runtime-fabric-release-notes.adoc[Anypoint Runtime Fabric Release Notes] for a complete list of supported and deprecated ciphers. 
```

Comment:
I think we should point to the Anypoint Security Edge release notes, not the RTF release notes.

Reply:
Got it - thanks!